deploy/examples/crds.yaml

```diff
@@ -346,10 +346,10 @@ spec:
                       description: Details contains potential status errors
                       type: string
                     lastChanged:
-                      description: LastChanged is the last time time the status last changed
```

Comment:
crds.yaml and resources.yaml are generated files, so we won't be able to update them unless the source is also fixed. Since these strings all look like they're pulled from k8s APIs, you'll just need to revert them for now. This is why the CI is failing, because the generated files don't match what's expected.